### DISCUSSION TITLE
Does Radioss have tiebreak card support?

### DISCUSSION THREAD

**Discussion by halfwitbm** [OpenRadioss/OpenRadioss#1901] [Q&A] [answered]
Hi All,

I have checked all the information on Radioss 2022.3 help, but I didn't find tie break information, only found tie information. Did someone know whether Radioss has the support for tie break mechanism? 

Thanks,

Bin

**Comment by MarianBulla** [comment]
Hi Bin,
thanks for your interest in all the features in Radioss and OpenRadioss.
Yes the fiebreak feature is available in Radioss and OpenRadioss since many years.
Just have a look the the interface TYPE2:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/a325d7f4-8fcd-4754-b1d6-44013052d7f8)

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by halfwitbm** [comment]
Thank you very much, Marian. I still have another question. Is this kind of tied-failed mechanism same as tiebreak in LS-DYNA card? I found for LS-DYNA *contact_automatic_surface_to_surface_tiebreak, the contact force has a transition region: from 0 to a value, then from this value to 0. something like following picture.  Does /inter/type2 have similar smooth contact force variation mechanism or contact force suddenly fall to 0 after spotweld fail?  Thank you.
![image](https://github.com/OpenRadioss/OpenRadioss/assets/46469179/118e26fe-32b2-4d23-b5d9-6c5b32e46d7e)

**Reply by MarianBulla** [reply]
Hi Bin,
Yes, you can define this "softening" behavior before the interface "breaks" and the tied nodes will be released:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/384bbeb9-a77a-44f0-83a1-ce7ac34f89a0)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/3a060087-3a02-4cab-b97f-c8d3ad0cd236)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/a493896e-8581-40cc-86ce-6b1610875d1a)

I don't know how it is realized in detail in LS-Dyna.

But it works well in OpenRadioss.

Hope it helps you ?

Thanks and best regards,
Marian

**Comment by halfwitbm** [comment]
Hi Marian. The question I really would like to ask is whether Radioss has surface-to-surface tie-break mechanism? Do we have to define a lot of spotwelds to achieve the surface-to-surface tie break mechanism? Thanks.

**Reply by MarianBulla** [reply, marked as answer]
Hi Bin,
there are a lot of different possibilites to model spotwelds:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/6a31645e-caca-4fc9-8824-20180a62ece4)

If you want to model single-point connections, concidering the connection-zone influence to the failure (which is possible), then you should go with mesh-independant cohesive-zone-element (as used in MAT59 or more advanced in MAT83). But it depends on your mesh-size and the type of connection, you would like to model (spotwelds/rivets/bolts/... or area-based connections, like adhesives).
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/671efec5-c0ef-41bf-bb7f-ccb24fc59b10)

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/245c7866-e267-4de1-aa9e-460a8b334477)

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/04797b41-76e7-4c68-a6d1-80f3ad045a8c)

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/1de0a5fc-0116-437e-a990-6015b11afc2d)

e.g.: If you are going to model single-point connections using a coarse mesh, then a good approach is to use mesh-independant placed beams, connected with the TYPE2 interface.

![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/39f103f5-6132-4336-a893-5149dd4c3ec0)

For larger connected areas with fine mesh, you can use the failure model within the TYPE2 interface. You can define tirectly a larger area (group of elements or entire parts), which are connected. You don't need to pick individuall nodes. This will be done by the solver, automatically.

A good pre-processor is able to prepare the input-dec in a user-friendly way.

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by halfwitbm** [reply]
Thank you very much for your detailed answer, Marian. It's a great help for me.

Best Regards,
Bin

**Reply by MarianBulla** [reply]
You're always welcome :-)

Thanks and best regards,
Marian
